app/views/shared/_navbar.html.erb

```diff
@@ -1,49 +1,51 @@
-<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+<nav class="navbar navbar-tripselector fixed-top" data-controller="navbar" data-navbar-target="navcontainer" data-action="scroll@window->navbar#update">
   <div class="container-fluid">
-
-      <%= link_to "/", class: "navbar-brand" do %>
-        <%= image_tag "logo.png", style:"width: 10%;", class:"m-0", id: "image_space" %>
-        <span class="navbar-text text-dark">
-            Trips Selector
-          </span>
-      <% end %>
-
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <%# start of left side %>
+          <%= link_to "/", class: "navbar-brand" do %>
+            <%= image_tag "tripselector-logo.svg", class:"navbar-logo" %>
+          <% end %>
+    <%# end of left side %>
```

Comment:
Logo is now an SVG. This allows it to change size depending on device and VP size, without becoming blurry.